extensions/ql-vscode/src/interface.ts

```diff
@@ -154,6 +154,18 @@ export class ResultsView extends AbstractWebview<IntoResultsViewMsg, FromResults
         this.navigatePathStep.bind(this, -1)
       )
     );
+    this.push(
+      commandRunner(
+        'codeQLQueryResults.nextAlert',
+        this.navigateAlert.bind(this, 1)
```

Comment:
Instead of making this a number, can you make this an enum:

```
enum NavDirection {
   Next = 1,
   Prev = -1
}
```

And you can change the messages above as well. You can put this in `interface-types.ts` and have it shared with the results view.

Reply:
There is `enum NavigationDirection` now